returnn/datasets/lm.py

```diff
@@ -132,7 +132,8 @@ def __init__(self,
       with open(orth_symbols_map_file, 'rb') as f:
         self.orth_symbols_map = pickle.load(f)
       self.orth_symbols = self.orth_symbols_map.keys()
-      self.labels["data"] = list(self.orth_symbols)
+      reverse_map = {i: sym for (sym, i) in self.orth_symbols_map.items()}
```

Comment:
I think we need a `sorted` here to have it deterministic on older Python versions.